src/ucs/config/parser.c

```diff
@@ -1003,10 +1002,20 @@ static void ucs_config_parser_mark_env_var_used(const char *name, int *added)
     }
 
     kh_put(ucs_config_env_vars, &ucs_config_parser_env_vars, key, &ret);
+    if (ret <= 0) {
```

Comment:
we can just print ucs_warn if ret <= 0

Reply:
then it will be a memory leak for `key` or I'm missing smth?

Reply:
sorry, i've meant ucs_warn instead of ucs_error